### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by rdnt**
- Removes the need to have replace directives, making `go install` functional again
- Bumps minimum go version to 1.17

### CODE REVIEW

File: pkg/composeutil/composeutil.go [hunk 1]
```diff
@@ -95,19 +97,19 @@ func ApplyPortMapping(p *types.Project, mapping PortConfigs) {
...
-			for _, cfg := range servicePortCfgs {
-				if int(port.Published) == cfg.OriginalPort && port.Protocol == cfg.Protocol {
-					port.Published = uint32(cfg.Port)
-					service.Ports[j] = port
-					p.Services[i] = service
-				}
-			}
-		}
-	}
+	//for i, service := range p.Services {
```
Comment (zuzuleinen) [inline pkg/composeutil/composeutil.go:100]: We no longer need this piece of code?

File: pkg/composeutil/composeutil.go [hunk 2]
```diff
@@ -166,19 +168,19 @@ func portConfigs(proj *types.Project, typ string) PortConfigs {
...
-			if spCfg.Protocol == typ {
-				if !PortAvailable(port, typ) && len(portServices[port]) == 0 {
-					portServices[port] = append(portServices[port], "host")
-				}
-
-				portServices[port] = append(portServices[port], s.Name)
-			}
-		}
-	}
+	//for _, s := range services {
```
Comment (zuzuleinen) [inline pkg/composeutil/composeutil.go:171]: Same as above

File: go.mod [hunk 3]
```diff
@@ -1,50 +1,189 @@
 module github.com/benchkram/bob
 
-go 1.16
+go 1.17
```
Comment (zuzuleinen) [inline go.mod:3]: Just curios, it doesn't work with go 1.18?

Comment (rdnt) [reply go.mod:3]: It does, but the minimum required is 1.17.

**Pull request merged**